CI/run-perf-ci-suite

```diff
@@ -650,6 +651,11 @@ function check_clusterbuster_option() {
     [[ -n "${known_clusterbuster_options[$option]:-}" ]]
 }
 
+function artifact_dirname() {
+    local basename=${1:-cb-ci-%T}
+    echo "${basename//%T/$(printf "%($snapshot_date_format)T" -1)}"
```

Comment:
I am curious what makes it different to do the following:
```
basename=${1:-cb-ci-T}
"${basename/%T/$(printf "%($snapshot_date_format)T" -1)}"
```

Reply:
If there are multiple occurrences of `%T` in the name they should all be expanded.